packages/sdk/CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ## Unreleased
 
-### Fix
+### Added
+
+- Support for using the SDK in a react native environment. Requires installing `react-native-get-random-values` peer dependency in consuming package.
```

Comment:
I guess this is slightly outdated now, since it requires more than that from the users, maybe just refer to the guide